source/OutfitterPanel.cpp

```diff
@@ -76,6 +76,12 @@ namespace {
 		}
 		return toRefill;
 	}
+
+	bool Imports(const map<CustomSale::SellType, CustomSale> &customSales, const Outfit *outfit)
+	{
+		return customSales.count(CustomSale::SellType::IMPORT) &&
+			customSales.at(CustomSale::SellType::IMPORT).Has(*outfit);
```

Comment:
I think `std::map::count()` does all the work of `std::map::find()` and then also checks if it's reached `std::map::end()`.
And, then, `std::map::at()` does everything `std::map::find()` does and then also dereferences where it ends up (throwing an exception if it's at `std::map::end()`).
So, just use `find()` and check the iterator against `end()`.